src/NuGet.Core/NuGet.Build.Tasks.Pack/NuGet.Build.Tasks.Pack.targets

```diff
@@ -142,27 +171,12 @@ Copyright (c) .NET Foundation. All rights reserved.
     <Delete Files="@(_PackageFilesToDelete)"/>
   </Target>
 
-  <Target Name="_CalculateInputsOutputsForPack" DependsOnTargets="_GetOutputItemsFromPack">
+  <Target Name="_CalculateInputsOutputsForPack"
+          DependsOnTargets="_GetAbsoluteOutputPathsForPack;_GetOutputItemsFromPack">
```

Comment:
You don't need this change.
_GetOutputItemsFromPack already depends on _GetAbsoluteOutputPathsForPack.